src/app/inputs/inputs.component.ts

```diff
@@ -1,44 +1,42 @@
-import { Component } from '@angular/core';
+import { Component, EventEmitter, Output } from '@angular/core';
 import { ApiService } from '../services/api.service';
-import { Task } from '../interface/task';
+import { Task } from '../interfaces/task.interface';
 
 @Component({
   selector: 'app-inputs',
   templateUrl: './inputs.component.html',
   styleUrls: ['./inputs.component.css']
 })
 export class InputsComponent {
-  
 
   constructor(private apiService: ApiService) { }
   
   addTask() {
-
     const inputTitle = document.getElementById('input-ul') as HTMLInputElement | null;
     const taskTitle = inputTitle?.value;
     const inputDesc = document.getElementById('desc-ul') as HTMLInputElement | null;
```

Comment:
Why did you use this kind of logic? We believe you should try to use a different logic to pick elements, as this might be prone to errors.

Maybe trying to use angular forms would be a way to solve this.